Text advanture game/Text advanture game/main.swift

```diff
@@ -0,0 +1,123 @@
+//
+//  main.swift
+//  Text advanture game
+//
+//  Created by Oscar Victoria Gonzalez  on 10/1/19.
+//  Copyright © 2019 Oscar Victoria Gonzalez . All rights reserved.
+//
+
+import Foundation
+
+let transportation1 = "4 train"
+let transportation2 = "2 train"
+
+let message = ("You've landed your first job interview as a software developer, you need to get there as soon as possible and your only form of transportation is the train. You have two choices. The \"4 train\" or the \"2 train\". Choose wisely.")
+
+print(message)
+
+
+
+let transportation  = readLine()!
```

Comment:
Force unwrapping is one of the many ways we can unwrap an optional type but it is not the best way and it is very unsafe. So nil coalescing or if let would be the best way to unwrap